docs/content/docs/glossary.md

```diff
@@ -11,6 +11,11 @@ The `@fluidframework/aqueduct` package is a library for building Fluid objects a
 Framework. Its goal is to provide a thin base layer over the existing Fluid Framework interfaces that allows developers
 to get started quickly. [Learn more.](https://fluidframework.com/docs/apis/aqueduct/)
 
+## Attached
+
+A Fluid container can be *attached* or *detached*. An attached container is connected to a Fluid service and can be
```

Comment:
maybe important to call out that once a container is attached, it can never be detached again. this is a one way transition